### DISCUSSION TITLE
Duplication of permissions

### DISCUSSION THREAD

**Discussion by karamokoisrael** [directus/directus#17118] [Archive]
### Describe the Bug

I was trying to create a simple permission manager using directus but I've noticed it was possible to create two permissions with the same payload. The problem is that when I tries to get the permissions back, I the same permission multiple times with only the id changing what is not convenient.

### To Reproduce

We just have to create two permissions with the same payload and try to get all added permissions in order to encounter the problem. 

### What version of Directus are you using?

9.22.1

### What version of Node.js are you using?

v18.12.1

### What database are you using?

Postgres

### What browser are you using?

Mozilla firefox

### How are you deploying Directus?

Docker

**Comment by rijkvanzanten** [comment]
This is currently working as expected. Just as with any other table, if you insert multiple records, multiple records will be returned on read.

**Reply by karamokoisrael** [reply]
I understand. I have written a custom hook to prevent multiple insertions for custom tables.
